docs/source/getting_started/install_developer.rst

```diff
@@ -263,12 +263,15 @@ prompt.
 Step 11: Install project requirements
 =================================================
 
-To install the project requirements, type the following commands in terminal:
+To install the project requirements, type the following commands in terminal from the project root folder (contains a file called
+``requirements.txt``):
 
 .. code-block:: none
 
-    sudo apt-get install libffi-dev
+    sudo apt-get install libffi-dev nodejs
```

Comment:
Required step for me: `sudo npm install gulp-cli -g` 
Should probably double check what the `-g` does.